components/molecules/MainHeading/main-heading.styled.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const StyledMainHeading = styled.header`
+    display: grid;
+    grid-template-columns: 3fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 0px 0px;
+    grid-template-areas:
+        'Text Emoji'
+        'job-title Emoji';
+    background-color: ${({ theme }) => theme.colours.background.one};
+    margin-top: 50px;
+
+    @media screen and (max-width: 700px) {
+        grid-template-areas:
+            'Text Text'
+            'job-title job-title'
+            'Emoji Emoji';
+    }
+`;
```

Comment:
This is just an example of taking away some of the boilerplate when accessing the theme

```suggestion
import styled, {css} from 'styled-components';

export const StyledMainHeading = styled.header`
    ${({theme: { colours }}) => {
        css`
             display: grid;
            grid-template-columns: 3fr 1fr;
            grid-template-rows: 1fr 1fr;
            gap: 0px 0px;
            grid-template-areas:
                'Text Emoji'
                'job-title Emoji';
            background-color: ${colours.background.one};
            margin-top: 50px;
        
            @media screen and (max-width: 700px) {
                grid-template-areas:
                    'Text Text'
                    'job-title job-title'
                    'Emoji Emoji';
            }
        `
    }}
`;
```

Reply:
that doesnt seem that much easier? altho i suppose if you had a few places needing colours you'd be repeating yourself less?

Reply:
Yeah basically, becomes more useful if you're using lots of values from a theme in one styled component.